src/Microsoft.Azure.ServiceBus/Constants.cs

```diff
@@ -40,5 +40,7 @@ static class Constants
         public static readonly int DefaultClientPumpPrefetchCount = 5;
 
         public static readonly TimeSpan NoMessageBackoffTimeSpan = TimeSpan.FromSeconds(5);
+
+        public static readonly TimeSpan MessageReceiverStartPumpInitialReceiveTimeout = TimeSpan.FromSeconds(1);
```

Comment:
👍 perhaps we should raise an issue about this so that at some point in time we can have a look at it and figure out what's going on?